tests/realtime/pubSub.js

```diff
@@ -67,6 +67,12 @@ describe('tests/realtime/pubsub.js >', () => {
     });
 
     it('subscribers receive all published messages', (done) => {
+      // No subscribers if running with separate real-time application
+      if (process.env.REALTIME_APPLICATION) {
```

Comment:
Does this actually get set when running the tests?

Reply:
This new script in `package.json`:

```
    "test-realtime-application": "REALTIME_APPLICATION=https://abc.def npm run test-with-coverage -- tests/realtime/pubSub.js",
```